draft-ietf-quic-transport.md

```diff
@@ -293,8 +307,8 @@ Long headers are used for packets that are sent prior to the completion of
 version negotiation and establishment of 1-RTT keys. Once both conditions are
 met, a sender switches to sending packets using the short header
 ({{short-header}}).  The long form allows for special packets - such as the
-Version Negotiation packet - to be represented in this uniform fixed-length
-packet format. Packets that use the long header contain the following fields:
+Version Negotiation packet - to be represented in this uniform packet format.
```

Comment:
It's actually weird to use version negotiation as an example here, because there's no packet number for version negotiation.  And I think there's no length either?